amivapi/tests/test_sentry.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+#
+# license: AGPLv3, see LICENSE for details. In addition we strongly encourage
+#          you to buy us beer if we meet and you like the software.
+"""Sentry integration tests."""
+
+from os import getenv
+import warnings
+
+from flask.signals import got_request_exception
+
+from amivapi.tests.utils import WebTestNoAuth, skip_if_false
+
+# Get test dsn from environment
+SENTRY_DSN = getenv('SENTRY_TEST_DSN')
+
+
+class SentryIntegrationTest(WebTestNoAuth):
+    """Raise an error to check if it gets sent to Sentry."""
+
+    def setUp(self, *args, **kwargs):
+        """Extended setUp.
+
+        Move environment variables to config.
+        """
```

Comment:
The first line of the docstring is useless. Use the second line as first line instead.